app/src/androidTest/java/com/github/se/cyrcle/model/parking/ParkingRepositoryFirestoreTest.kt

```diff
@@ -0,0 +1,158 @@
+package com.github.se.cyrcle.model.parking
+
+import android.content.Context
+import android.util.Log
+import androidx.test.core.app.ApplicationProvider
+import com.google.firebase.FirebaseApp
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.firestore.firestoreSettings
+import com.google.firebase.firestore.memoryCacheSettings
+import com.google.firebase.firestore.persistentCacheSettings
+import java.util.concurrent.CountDownLatch
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.tasks.await
+import kotlinx.coroutines.test.runTest
+import org.junit.Assert.fail
+import org.junit.Before
+import org.junit.Test
+
+class ParkingRepositoryFirestoreTest {
+
+  private lateinit var parkingRepositoryFirestore: ParkingRepositoryFirestore
+
+  @Before
+  fun setUp() = runTest {
+    val context = ApplicationProvider.getApplicationContext<Context>()
+    FirebaseApp.initializeApp(context)
+    Log.d("AHHHHH", "Starting Before")
```

Comment:
I'm assuming this was a debugging line which can be removed?

Reply:
Yep! Good spot! I dealt with that